pkg/resources/digitalocean/dns/dns.go

```diff
@@ -17,15 +17,14 @@ limitations under the License.
 package dns
 
 import (
+	"context"
 	"errors"
 	"fmt"
+	"github.com/digitalocean/godo"
```

Comment:
style note: usually when I am coding golang, the internal libraries are imported first. Then rowchange and after that other imports. Like in this case I recommend moving godo after go internal imports

Reply:
Sure, updated.